sites/teams.html

```diff
@@ -104,4 +104,160 @@ <h1 class="main-header">
     pics from <em>https://www.futhead.com/card-generator/</em>
 </footer>
 </body>
+<script type="text/javascript">
+
+    let italy_teams = [
+        "juventus",
+        "napoli",
+        "parma",
+        "inter"
+    ]
+    let poland_teams = [
+        "gornik",
+        "piast",
+        "wisla",
+        "lech"
+    ]
+    let germany_teams = [
+        "bayern",
+        "borussia",
+        "hertha",
+        "eintracht"
+    ]
+    let england_teams = [
+        "liverpool",
+        "westham",
+        "wolves",
+        "man utd"
+    ]
+
+    let country_leagues = {
+        "italy": italy_teams,
+        "poland": poland_teams,
+        "germany": germany_teams,
+        "england": england_teams
+    };
```

Comment:
Structure of this date should look more like this
```

const data = {
  premierLeague: {
    teams: ['Chelsea', 'Liverpool'],
    
  },
  laLiga: {},
  bundesliga: {},
};
```

It will help keep clean and maintainable code